tutorials/io/data_paths.rst

```diff
@@ -65,21 +66,25 @@ Editor data paths
 The editor uses different paths for user data, user settings, and cache,
 depending on the platform. By default, these paths are:
 
-+---------------+---------------------------------------------------+
-| Type          | Location                                          |
-+===============+===================================================+
-| User data     | - Windows: ``%APPDATA%\Godot\``                   |
-|               | - macOS: ``~/Library/Application Support/Godot/`` |
-|               | - Linux: ``~/.local/share/godot/``                |
-+---------------+---------------------------------------------------+
-| User settings | - Windows: ``%APPDATA%\Godot\``                   |
-|               | - macOS: ``~/Library/Application Support/Godot/`` |
-|               | - Linux: ``~/.config/godot/``                     |
-+---------------+---------------------------------------------------+
-| Cache         | - Windows: ``%TEMP%\Godot\``                      |
-|               | - macOS: ``~/Library/Caches/Godot/``              |
-|               | - Linux: ``~/.cache/godot/``                      |
-+---------------+---------------------------------------------------+
++------------------------------+-----------------------------------------------------------+
+| Type                         | Location                                                  |
++==============================+===========================================================+
+| User data                    | - Windows: ``%APPDATA%\Godot\app_userdata\``              |
+|                              | - macOS: ``~/Library/Application Support/Godot/``         |
+|                              | - Linux: ``~/.local/share/godot/``                        |
++------------------------------+-----------------------------------------------------------+
+| User Data                    | - Windows: ``%APPDATA%\``                                 |
```

Comment:
`%APPDATA%\[custom_name]` or `%APPDATA%\[custom_dir]` would be a bit clearer, don't you think?